test/core/dialects/snowflake_test.py

```diff
@@ -111,6 +112,29 @@
             SemiStructuredAccessorSegment,
             "SELECT ID :: VARCHAR as id, OBJ : userId :: VARCHAR as user_id from x",
         ),
+        (
```

Comment:
Would you mind changing all these tests to follow the .sql and .yml file format in here? https://github.com/sqlfluff/sqlfluff/tree/master/test/fixtures/parser

Reply:
Done. Made a note to clean up the rest of them too (will submit a separate PR for this once this gets merged to avoid conflicts).